README.md

```diff
@@ -531,6 +531,34 @@ Not implemented yet, track it in story https://github.com/strongloop/loopback-co
 -  [LoopBack query](http://loopback.io/doc/en/lb3/Querying-data.html) support for: fields, limit, order, skip and where filters.
 - Please check [Advanced Queries](https://github.com/strongloop/loopback-connector-cloudant/blob/master/doc/advanced-queries.md) for details about regex filter, nested filter and order.
 
+# View
+
+Given a design doc name and the view name in it, user can use a connector level function `viewDocs` to query the view.
+
+Since `viewDocs` is a specific api for Cloudant connector only, it is not attached to the dataSource Object defined in loopback-datasource-juggler, which means the correct way to call it is `ds.connector.viewDocs`:
+
+*/server/script.js*
+```javascript
+module.exports = function(server, cb) {
+  // Get Cloudant dataSource as `ds`
+  // 'cloudantDB' is the name of Cloudant datasource created in 
+  // 'server/datasources.json' file
+  var ds = server.datasources.cloudantDB;
+
+
+  ds.on('connected', function() {
+    // 1. Please note `ds.connector.viewDocs()` is the correct way to call it,
+    // NOT `ds.viewDocs()`
+    // 2. This api matches the Cloudant endpoint:
+    // GET /db/_design/<design-doc>/_view/<view-name>
+    ds.connector.viewDocs('design_doc', 'view_name', cb);
```

Comment:
maybe expand `cb` to be inline to show what we expect to see in the callback function like `function (err, views)`?